docs/docs/lists-conditional-rendering.md

```diff
@@ -0,0 +1,264 @@
+
+The highlighted lines below show where `loginControlButton` is created and set to a default value of "`<LoginButton />`", then where it conditionally gets changed depending on current state, and finally where `{loginControlButton}` is rendered.
+
+```javascript{21,23,27}
+function LogoutButton(props) {
+  return <button onClick={props.click}>Logout</button>
+}
+function LoginButton(props) {
+  return <button onClick={props.click}>Login</button>
+}
+class LoginControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {loggedIn: true}
+    this.login = this.login.bind(this);
+    this.logout = this.logout.bind(this);
+  }
+  login() {
+    this.setState({loggedIn: true,});
+  }
+  logout() {
+    this.setState({loggedIn: false});
+  }
+  render() {
+    var loginControlButton = <LoginButton click={this.login} />;
+    if (this.state.loggedIn) {
+      loginControlButton = <LogoutButton click={this.logout} />;
+    }
+    return (
+      <div>
+        {loginControlButton}
+      </div>
+    );
+  }
+}
+
+```
+
+<a target="_blank" href="https://codepen.io/ericnakagawa/pen/Egpdrz?editors=0010">Try it out on Codepen.</a>
+
+
+### Prevent Component Rendering
+
+In some cases, you will not want a component to render. To prevent components from rendering, return `null` or `false` from the `render()` function.
+
+In the example below, a `<Page />` component renders a child `<WarningBanner />` element and a button. The button calls `this.toggleWarning` to switch the value of `this.state.showWarning`. If the `this.state.showWarning` is false, then nothing will be rendered.
+
+The highlighted lines below show where the variable `warningBanner` is set to a default value of null and then where it is rendered in `return()`.
+
+```javascript{17,23}
+function WarningBanner(props) {
+  return <div className="warning">Warning!</div>
+}
+class Page extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {showWarning: true, toggleText: "hide"}
+    this.toggleWarning = this.toggleWarning.bind(this);
+  }
+  toggleWarning() {
+    this.setState({
+      showWarning: !this.state.showWarning,
+      toggleText: !this.state.showWarning?"hide":"show"
+    });
+  }
+  render() {
+    let warningBanner = null;
+    if (this.state.showWarning) {
+      warningBanner = <WarningBanner />;
+    }
+    return (
+      <div>
+        {warningBanner}
+        <button onClick={this.toggleWarning}>{this.state.toggleText}</button>
+      </div>
+    );
+  }
+}
+```
+
+<a target="_blank" href="https://codepen.io/ericnakagawa/pen/ozragV?editors=0011#0">Try it out on Codepen.</a>
+
+### Rendering Multiple Components Using {}
+
+You can build collections of components and include them in <a target="_blank" href="/react/docs/introducing-jsx.html">JSX</a> with curly braces `{}` just as you would embed values with Javascript.
```

Comment:
"collections of elements", not components.
"Rendering Multiple Components" is fine because you get components as a result, but whenever we talk about manipulating JSX we always talk about the elements.